infra/gcp/terraform/k8s-infra-public-pii/main.tf

```diff
@@ -99,6 +112,18 @@ data "google_iam_policy" "audit_logs_gcs_bindings" {
       "group:cloud-storage-analytics@google.com",
     ]
   }
+  binding {
+    role = "roles/storage.legacyBucketReader"
+    members = [
+      "serviceAccount:${google_service_account.public_pii_asn_etl.email}",
+    ]
+  }
+  binding {
+    role = "roles/storage.legacyBucketReader"
+    members = [
+      "serviceAccount:${data.google_service_account.ii_sandbox_asn_etl.email}",
+    ]
+  }
```

Comment:
I'm not sure to understand by why do we need to do this binding ? Is it possible to have something that explain the different interactions between BigQuery and Google Cloud Storage ?

Reply:
The Pii data is in a bucket named _k8s-infra-artifacts-gcslogs_ in the _public_pii_ project.
In order to run the job to generate up to date data to display for the image/artifact spending DataStudio report, the logs inside of the bucket are needed.
The ServiceAccount that the job runs as needs to have access to the bucket mentioned.
After discussing with @spiffxp and @hh, I believe these are the right lines to add.

Reply:
SGTM